<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajorityVoting</class>
 <widget class="QDockWidget" name="MajorityVoting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>123</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>123</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>Majority Voting</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Min. Diff. %</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="lbMaxPixelEntropy">
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="slMaxPixelEntropy">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Min. Ratio</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="slMinRatio">
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="lbMinRatio">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pbMinAbsPercent_Plot">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pbMaxPixelEntropy_Plot">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lbMinAbsPercent">
      <property name="text">
       <string>100 %</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lbMinDiffPercent">
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pbMinRatio_Plot">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Max. Pixel Ent.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="slAbsMinPercent">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="slMinDiffPercent">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pbMinDiffPercent_Plot">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Abs. Min. %</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pbSample">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Sample Maj. Voting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
